.github/workflows/ci.yml

```diff
@@ -1,61 +1,25 @@
 name: CI
-
 on:
-  push:
-    branches: [ master ]
   pull_request:
-    branches: [ master ]
-
 jobs:
-  build:
-
+  build-and-deploy:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [18.x]
-
+    permissions: write-all
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm install
-    - run: npm run build
-    - run: npm test
-  DeployPages:
-    runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'push' }}
-    steps:
-      - name: Checkout 🛎️
-        uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+      - name: Checkout repository
+        uses: actions/checkout@master
+      - name: Install pnpm
+        run: npm i -g pnpm
+      - run: pnpm install
+      - run: pnpm lint
+      - run: pnpm check-build
+      - run: nohup pnpm prod-start &
+      - run: nohup node cypress/minecraft-server.mjs &
+      - uses: cypress-io/github-action@v5
         with:
-          persist-credentials: false
-          fetch-depth: 0
-      - name: Edit config
-        run: |
-          sed -i -E 's/^  "defaultProxy": ""/  "defaultProxy": "pproxy.rom1504.fr"/g' config.json
```

Comment:
@rom1504 why isn't this proxy used by default here? I mean why do we have to edit the config here currently ?

Reply:
This is to edit the config to replace by the default

Reply:
The web demo here is using this proxy but it is not particularly recommended for other users to use this proxy

Reply:
> The web demo here is using this proxy but it is not particularly recommended for other users to use this proxy

This actually answers my question, I forgot that the config file is deployed to npm, ty. I just wondered if I could use it on my web demo, but now I have my own proxy which is hosted on free webhost.